From 1744159180ac94a72036d7232e674c58cedee70a Mon Sep 17 00:00:00 2001
From: Libin Yang <libin.yang@linux.intel.com>
Date: Wed, 11 Dec 2019 14:07:19 +0800
Subject: [PATCH 6/6] ucm: setup conf_format after getting ALSA_CONFIG_UCM_VAR
 successfully

Set the conf_format to 1 after getting ALSA_CONFIG_UCM_VAR successfully.
Otherwise, the conf_format is not set in this scenario.

Signed-off-by: Libin Yang <libin.yang@linux.intel.com>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---
 src/ucm/parser.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/ucm/parser.c b/src/ucm/parser.c
index ba500277..d61124a3 100644
--- a/src/ucm/parser.c
+++ b/src/ucm/parser.c
@@ -89,6 +89,8 @@ static void configuration_filename(snd_use_case_mgr_t *uc_mgr,
 		env = getenv(ALSA_CONFIG_UCM2_VAR);
 		if (env == NULL) {
 			env = getenv(ALSA_CONFIG_UCM_VAR);
+			if (env)
+				uc_mgr->conf_format = 1;
 		} else {
 			uc_mgr->conf_format = 2;
 		}
-- 
2.24.1

